### PR TITLE
Format all code in project

### PR DESCRIPTION
**Pull request by austinkelleher**
Two commits:

1. Format all code in project
2. Update top-level `package.json` to include `format:check` command and check after tests have run

### CODE REVIEW

File: .prettierignore [hunk 1]
```diff
@@ -1,3 +1,5 @@
 coverage/
 work/
-dist/
\ No newline at end of file
+dist/
+.j1-integration
+.j1-integration-cache
```
Comment (austinkelleher) [inline .prettierignore:5]: We don't want to format these files

File: .gitattributes [hunk 2]
```diff
@@ -0,0 +1,6 @@
+# Ensure that Windows users are using LF line endings. GitHub action runners
+# respect properties of the `.gitattributes` file. Without this, 
+# `prettier --check` will fail on Windows.
+# 
+# See: https://prettier.io/docs/en/options.html#end-of-line
+* text=auto eol=lf
```
Comment (austinkelleher) [inline .gitattributes:6]: Needed to add this to support `prettier --check` on Windows. See comment.

File: .prettierignore (hunk 1 above)
Comment (austinkelleher) [inline .prettierignore:5]: These directories should not be formatted.

File: package.json [hunk 3]
```diff
@@ -6,15 +6,16 @@
   ],
   "scripts": {
     "lint": "eslint --ext .ts,.tsx,.js,.jsx --cache .",
-    "test": "jest && yarn --cwd ./packages/integration-sdk-runtime test:event-handlers",
+    "test": "jest && yarn --cwd ./packages/integration-sdk-runtime test:event-handlers && yarn format:check",
```
Comment (austinkelleher) [inline package.json:9]: Format check after running the tests

File: package.json [hunk 4]
```diff
@@ -6,15 +6,16 @@
...
     "test:ci": "yarn lint && yarn build --force && yarn test",
     "clean": "yarn lerna clean --yes && yarn clean:build && rm -r node_modules",
     "clean:build": "yarn lerna exec \"rm -rf ./dist tsconfig.tsbuildinfo\"",
     "prebuild": "yarn clean:build",
     "build": "tsc -b packages/integration-sdk-core packages/integration-sdk-runtime packages/integration-sdk-private-test-utils packages/integration-sdk-cli packages/integration-sdk-testing",
     "prebuild:dist": "yarn clean:build",
     "build:dist": "lerna run build:dist",
-    "format": "prettier --write '**/*.{js,ts,md,json}'",
+    "format": "prettier --write \"**/*.{ts,js,json,md,yml}\"",
+    "format:check": "prettier --check \"**/*.{ts,js,json,md,yml}\"",
```
Comment (austinkelleher) [inline package.json:18]: New command

File: package.json [hunk 5]
```diff
@@ -6,15 +6,16 @@
...
+    "test": "jest && yarn --cwd ./packages/integration-sdk-runtime test:event-handlers && yarn format:check",
     "test:ci": "yarn lint && yarn build --force && yarn test",
     "clean": "yarn lerna clean --yes && yarn clean:build && rm -r node_modules",
     "clean:build": "yarn lerna exec \"rm -rf ./dist tsconfig.tsbuildinfo\"",
     "prebuild": "yarn clean:build",
     "build": "tsc -b packages/integration-sdk-core packages/integration-sdk-runtime packages/integration-sdk-private-test-utils packages/integration-sdk-cli packages/integration-sdk-testing",
     "prebuild:dist": "yarn clean:build",
     "build:dist": "lerna run build:dist",
-    "format": "prettier --write '**/*.{js,ts,md,json}'",
+    "format": "prettier --write \"**/*.{ts,js,json,md,yml}\"",
```
Comment (austinkelleher) [inline package.json:17]: Escape double quotes instead of single quotes to support Windows.

**Review by aiwilliams — APPROVED** [review]

**Comment by aiwilliams** [comment]
You could have done this as 100 commits to improve your public profile. Missed opportunity bro.

**Pull request merged**